tock/tock/urls.py

```diff
@@ -39,6 +39,9 @@ def check_if_staff(user):
 
     path('auth/', include('uaa_client.urls')),
 
+    path('session-warning', tock.views.session_warning, name="SessionWarning"),
```

Comment:
this seemed like an okay place to put these endpoints but ... i dunno! is there a better one?